internal/metamorph/processor.go

```diff
@@ -492,8 +526,19 @@ func (p *Processor) statusUpdateWithCallback(ctx context.Context, statusUpdates,
 		updatedData = append(updatedData, updatedDoubleSpendData...)
 	}
 
+	statusHistoryUpdates := filterUpdates(statusUpdates, updatedData)
+	shUpdatedData, err := p.store.UpdateStatusHistoryBulk(ctx, statusHistoryUpdates)
+	if err != nil {
+		return err
```

Comment:
I think if the update of status history fails, we shouldn't return an error, but just log it.

This feature just provides additional information for each transaction, but is not critical

Reply:
Done